### PR TITLE
Stub out api endpoints

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler, @hodgestar depends on #23, otherwise ready for review

**Comment by justinvdm** [comment]
Ready for review.

File: tests/utils.js [hunk 1]
```diff
@@ -10,6 +13,14 @@ function captureError(fn) {
 }
 
 
+function restore(obj) {
+  each(obj, v => {
+    if (v && v.toString() === 'stub') v.restore();
+  });
+}
```
Comment (hodgestar) [inline tests/utils.js]: Is this function being used anywhere?

**Comment by hodgestar** [comment]
Left one question, otherwise looks good.

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**